src/security-council-mgmt/SecurityCouncilUpgradeExecutor.sol

```diff
@@ -69,6 +61,13 @@ contract SecurityCouncilUpgradeExecutor is
             }
         }
 
+        for (uint256 i = 0; i < _membersToAdd.length; i++) {
```

Comment:
could removing before adding cause the safe to revert because number of members is below threshold?

Reply:
https://github.com/safe-global/safe-contracts/blob/6434676c47fe50fa0d21f56b26747aa1216504ef/contracts/base/OwnerManager.sol#L78